recipes/nss/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+cd nss
+
+mkdir -p "${PREFIX}/bin"
+mkdir -p "${PREFIX}/lib"
+
+make    \
+    USE_64=1 \
+    PREFIX="${PREFIX}" \
+    NSPR_PREFIX="${PREFIX}" \
+    NSPR_INCLUDE_DIR="${PREFIX}/include/nspr" \
+    C_INCLUDE_PATH="${PREFIX}/include:${PREFIX}/include/nspr" \
+    LIBRARY_PATH="${PREFIX}/lib" \
+    USE_SYSTEM_ZLIB=1 \
+    NSS_USE_SYSTEM_SQLITE=1 \
+    NS_USE_GCC=1 \
+    NSDISTMODE='copy' \
+    all latest
+
+cd ../dist
+FOLDER=$(<latest)
+cd $FOLDER
+
+cp -rL bin "${PREFIX}"
+cp -rL lib "${PREFIX}"
```

Comment:
Is there no `make install`?

Reply:
I tried that but it didn't work.. I search in the documentation and they tell you to manually install that. That was the reason why I did that "by hand".

"The build system of NSS originated from Netscape's build system, which predated the "configure; make; make test; make install" sequence that we're familiar with now. Our makefiles also have an "install" target, but it has a different meaning: our "install" means installing the headers, libraries, and programs in the appropriate directories under mozilla/dist. "

https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/Reference/Building_and_installing_NSS/Installation_guide

But for me that is still fuzzy, however if you know a better solution for that, please let me know and I will try! :)

Reply:
Sure, I will take a look! Thanks!